frontend/src/metabase/components/Popover/Popover.jsx

```diff
@@ -201,11 +202,12 @@ export default class Popover extends Component {
       this._tether.destroy();
       delete this._tether;
     }
+
     if (this._popoverElement) {
-      ReactDOM.unmountComponentAtNode(this._popoverElement);
```

Comment:
I'm not 100% sure about it, but we do not add any event listener manually and seems remove this element from DOM correctly

Reply:
I believe `unmountComponentAtNode` should be only used in pair with `ReactDOM.render`, but it's not called here

Reply:
Maybe previously there was no `createPortal` call but `ReactDOM.render` instead, and this is a leftover

Reply:
nice catch!